src/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -38,13 +38,51 @@ trade-offs of pre-warming an engine.
 ### Create a FlutterEngine
 
 The proper place to create a `FlutterEngine` is specific
-to your host app. As an example, we demonstrate creating a
-`FlutterEngine`, exposed as a property, on app startup in
-the app delegate.
+to your host app. 
 
 {% samplecode engine %}
 
-{% sample Swift %}
+{% sample SwiftUI %}
+As an example, we demonstrate creating a
+`FlutterEngine` inside an `ObservableObject`, 
+and passing that into a `View` using the 
+ `environmentObject()` modifier. 
+
+ **In `MyApp.swift`:**
+ <!--code-excerpt "MyApp.swift" title-->
+ ```swift
+import SwiftUI
+import Flutter
+// Used to connect plugins (only if you have plugins with iOS platform code).
+import FlutterPluginRegistrant
```

Comment:
**question:** Which line does this modify? The previous or the following? It reads like it's the previous. If it is the `FlutterPluginRegistrant`, then move this comment to after that line.

**suggestion:** Consider the context. _What_ is used to connect plugins? Also, why use the parenthetical? It can be combined.

```suggestion
import Flutter
import FlutterPluginRegistrant
// The previous library connects plugins with iOS platform code to this app.
```

Reply:
Hmm I'm more familiar with comments that go before the line that it is describing, but I can make it explicit that we are referring to FlutterPluginRegistrant